backend/src/main/resources/application.yaml

```diff
@@ -1,6 +1,11 @@
 keycloak:
-  admin-api-url: https://common-logon-dev.hlth.gov.bc.ca/auth/admin/realms/moh_applications
-  base-oauth-url: https://common-logon-dev.hlth.gov.bc.ca/auth/realms/moh_applications
+  realm: moh_applications
+  auth-server-url: https://common-logon-dev.hlth.gov.bc.ca/auth/
+  bearer-only: true
+  use-resource-role-mappings: true
+  resource: user-management
+config:
+  base-oauth-url: https://common-logon-dev.hlth.gov.bc.ca/auth/admin/realms/moh_applications
```

Comment:
Suggest leaving this as admin-api-url, it's not use for authentication it's actually the base url for keycloaks admin API endpoint

Reply:
Yes, you're right. I just changed this when I was debugging something and getting a headache.